doc/dox_comments/header_files/ssl.h

```diff
@@ -1631,9 +1631,40 @@ int  wolfSSL_set_fd (WOLFSSL* ssl, int fd);
     \sa wolfSSL_CTX_SetIORecv
     \sa wolfSSL_SetIOReadCtx
     \sa wolfSSL_SetIOWriteCtx
+    \sa wolfDTLS_SetChGoodCb
 */
 int wolfSSL_set_dtls_fd_connected(WOLFSSL* ssl, int fd)
 
+/*!
+    \ingroup Setup
+
+    \brief Allows setting a callback for DTLS client hello "good".
```

Comment:
it should be mentioned that if HRR Cookie is enabled, this CB is triggered only for CH with a valid cookie.

Reply:
Added description.